php/libraries/Utility.class.inc

```diff
@@ -427,9 +427,10 @@ class Utility
      *
      * @param mixed $var The variable to be converted to an array.
      *
-     * @return array If $var is an array, var, otherwise an array containing $var
+     * @return array|string If $var is an array, $var,
+     *                          otherwise an array containing $var
      */
-    static function toArray($var): array
+    static function toArray($var)
```

Comment:
if $var is a string, the same string will be returned! 
```if (is_array($var) ...)``` will be false skipping to line 438 ```return $var```

Reply:
Yes, that's intentional.

Reply:
In that case, the docblock do not reflect the work done by this function.

Reply:
>      * Note: This does not change a string into an array with one string element

> @return array|string If $var is an array, $var,
     *                          otherwise an array containing $var

That's how I found out that this is intentional behaviour.

Reply:
Note that this PR is just a tiny bugfix and that #4151 addresses concerns about this confusing function.